### PR TITLE
DOCS-8624-Scripting-examples-edits-kt

### PR DESCRIPTION
**Pull request by kevintroller**
Scripting 2.0 Examples doc

### CODE REVIEW

File: scripting/2.0/modules/ROOT/pages/edits/scripting-module-examples.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
+= Scripting Module - Examples - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Scripting Module examples help you use binding variables and access the registry.
```
Comment (kbostrom) [inline scripting/2.0/modules/ROOT/pages/edits/scripting-module-examples.adoc]:
```suggestion
The following Scripting Module examples show you how to use binding variables and how to access the registry.
```

File: scripting/2.0/modules/ROOT/pages/edits/scripting-module-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+== Use Binding Variables Example
+
+The following Scripting Module XML example shows how to:
+
+* Use a flow variable in the scripting code, such as `vars.increment`
+* Use scripting parameters by referencing them, such as `initialValue`
+* Log some information by using the `log` variable, such as `log.info("....")`
+* Use the `result` variable to set the result of the execution.
```
Comment (kbostrom) [inline scripting/2.0/modules/ROOT/pages/edits/scripting-module-examples.adoc]:
```suggestion
* Use the `result` variable to set the result of the execution
```

File: scripting/2.0/modules/ROOT/pages/edits/scripting-module-examples.adoc [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    </scripting:code>
+    <scripting:parameters><![CDATA[#[{ initialValue: 10 }]]]></scripting:parameters>
+</scripting:execute>
+----
+
+== Access the Registry Example
+The following Scripting Module XML example shows how to:
+
+* Access the registry
+* Make a flow stop or start
```
Comment (kbostrom) [inline scripting/2.0/modules/ROOT/pages/edits/scripting-module-examples.adoc]:
```suggestion
* Stop or start a flow
```

**Review by kbostrom — DISMISSED** [review]
Edited

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**